xc7/make/arch_define.cmake

```diff
@@ -56,6 +56,15 @@ function(ADD_XC7_ARCH_DEFINE)
         --blif \${OUT_EBLIF} \
         --pcf \${INPUT_IO_FILE} \
         --net \${OUT_NET}"
+    PLACE_CONSTR_TOOL
+      ${symbiflow-arch-defs_SOURCE_DIR}/xc7/utils/prjxray_create_place_constraints.py
+    PLACE_CONSTR_TOOL_CMD "${CMAKE_COMMAND} -E env \
+    PYTHONPATH=${symbiflow-arch-defs_SOURCE_DIR}/utils \
+    \${PYTHON3} \${PLACE_CONSTR_TOOL} \
+        --net \${OUT_NET} \
+        --input /dev/stdin \
+        --output /dev/stdout \
+        --connection_database ${CMAKE_CURRENT_BINARY_DIR}/channels.db"
```

Comment:
This path to the connection database won't work.  See how `FASM_TO_BIT_EXTRA_ARGS` work.

Reply:
Done